Apps/WebClient/src/ClientApp/src/components/timeline/filters.vue

```diff
@@ -108,7 +108,7 @@ export default class FilterComponent extends Vue {
         this.filters[4].isEnabled = this.config.modules["Note"];
         this.selectedFilters = [];
 
-        this.eventBus.$on("filterSelected", (filterName: string) => {
+        this.eventBus.$on("filter-selected", (filterName: string) => {
```

Comment:
good catch. There is a static class that holds the name of the messages "EventMessageName" please add that on there too so we don't have to hardcode the string names.
The one below will also need an update
```
private onCovidSubmit() {
        this.eventBus.$emit("filter-selected", "laboratory");
    }
```